platform/docs/docs/migration-guide/3p8-to-3p9/7-UI.md

```diff
@@ -147,3 +144,8 @@ All the DropDown and Icons are now in the @ohif/ui-next package.
 
 
 ---
+
+
+## ui, ui-next configs
```

Comment:
Also, if you add custom icons, you may need to add them using a new addIcon utility which adds the icon to both ui and ui-next